src/others/vg_lite_tvg/vg_lite_tvg.cpp

```diff
@@ -2418,4 +2435,86 @@ static void get_format_bytes(vg_lite_buffer_format_t format,
             break;
     }
 }
+
+static vg_lite_fpoint_t matrix_transform_point(const vg_lite_matrix_t * matrix, const vg_lite_fpoint_t * point)
+{
+    vg_lite_fpoint_t p;
+    p.x = (vg_lite_float_t)(point->x * matrix->m[0][0] + point->y * matrix->m[0][1] + matrix->m[0][2]);
+    p.y = (vg_lite_float_t)(point->x * matrix->m[1][0] + point->y * matrix->m[1][1] + matrix->m[1][2]);
+    return p;
+}
+
+static bool vg_lite_matrix_inverse(vg_lite_matrix_t * result, const vg_lite_matrix_t * matrix)
+{
```

Comment:
You can split this function into four functions for reuse elsewhere, like:
1. void vg_lite_matrix_identity(vg_lite_matrix_t *) 
2. float vg_lite_matrix_determinant(vg_lite_matrix_t *)
2. bool vg_lite_matrix_is_determinant(vg_lite_matrix_t *)
4. bool vg_lite_matrix_inverse(vg_lite_matrix_t *, vg_lite_matrix_t *)